src/qibocal/protocols/resonator_utils.py

```diff
@@ -0,0 +1,360 @@
+import numpy as np
+from numpy.typing import NDArray
+from scipy.ndimage import gaussian_filter1d
+from scipy.optimize import leastsq, newton
+
+
+def get_cable_delay(frequencies: NDArray, z: NDArray, num_points: int = 10) -> float:
+    """
+    Evaluates the cable delay τ caused by the length of the cable and the finite speed of light.
+    Performs a first-grade polynomial fit of the phase and extracts the angular coefficient.
+
+        Args:
+            frequencies (NDArray[float]): frequencies (Hz) at which the measurement was taken.
+            z (NDArray[complex]): S21 scattering matrix element.
+            num_points (int): number of points selected from both the start and the end of the
+                              frequencies array to perform the linear fit.
+
+        Returns:
+            The value (float) of the cable delay τ in seconds.
```

Comment:
Python supporting Unicode is definitely a nice feature. Though, it is very annoying to type.

However, for math symbols, we already have a more reliable alternative:
```suggestion
            The value (float) of the cable delay :math:`\tau` in seconds.
```